src/BreezyCore.php

```diff
@@ -101,9 +101,15 @@ public function boot(Panel $panel): void
             }
 
             if ($this->myProfile['shouldRegisterUserMenu']) {
-                $panel->userMenuItems([
-                    'account' => MenuItem::make()->url(Pages\MyProfilePage::getUrl()),
-                ]);
+                if(request()->route()->parameter('tenant') !=null){
+                    $panel->userMenuItems([
+                        'account' => MenuItem::make()->url( Pages\MyProfilePage::getUrl(panel: $panel->getId(),tenant: app($panel->getTenantModel())::where($panel->getTenantSlugAttribute()??'id',request()->route()->parameter('tenant'))->first())),
+                    ]);
+                }else{
+                    $panel->userMenuItems([
+                        'account' => MenuItem::make()->url( Pages\MyProfilePage::getUrl()),
+                    ]);
+                }
```

Comment:
```suggestion
                $panel->userMenuItems([
                    'account' => MenuItem::make()->url(fn () => Pages\MyProfilePage::getUrl()),
                ]);
```

Using a closure resolves the problem.

Reply:
@jeffgreco13 Sorry for pinging you in a merged pull request comment, but can you apply this fix instead of the one you merged? This one actually fixes the problem AND makes the link clickable. The merged solution does not make the link clickable for me in a tenant panel.